test/integration/02-user-wallet/02-tx-onchain-fees.spec.ts

```diff
@@ -0,0 +1,84 @@
+import * as Wallets from "@app/wallets"
+import { getOnChainWalletConfig } from "@config/app"
+import { InsufficientBalanceError, LessThanDustThresholdError } from "@domain/errors"
+import { bitcoindClient, bitcoindOutside, getUserWallet } from "test/helpers"
+
+jest.mock("@services/realtime-price", () => require("test/mocks/realtime-price"))
+jest.mock("@services/phone-provider", () => require("test/mocks/phone-provider"))
+
+const defaultAmount = 6000 as Satoshis
```

Comment:
We've been using this pattern in different places
```suggestion
const defaultAmount = toSats(6000)
```